CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Release Notes
+## 2.0.0
+
+- no changes on the code itself. We move the publishment from dockehub to ghcr.
```

Comment:
typos here

Reply:
> no changes on the code itself. We now publish the container on ghcr instead of docker hub.

Reply:
also add a note that we officially move it out of beta, we've been running 2.0 without changes in production for a long while